### PR TITLE
Some Captions Lab improvements #1169

### LINKED ISSUE #1169: Some Captions Lab improvements
Currently, when a slide has no notes and the Captions Lab button is clicked, nothing happens. This might be confusing for the user.

I suggest the following improvements:

If the Speaker Notes window is minimized, open it (since some users may not know where to find this also)
If there are no notes and the button is clicked, inform the user with a messagebox

### PR DESCRIPTION
**Pull request by Kenneth-LJS**
Fixes #1169

Improvements:
- Will show message when no slides are selected to apply captions
- Will show message and open the notes when the slide selected does not have captions
- Will NOT show message when there are multiple slides selected (users might select all slides and batch-apply captions)

### CODE REVIEW

**Comment by Kenneth-LJS** [comment]
Conflicts resolved

**Comment by Kenneth-LJS** [comment]
Conflicts resolved

File: PowerPointLabs/PowerPointLabs/NotesToCaptions.cs [hunk 1]
```diff
@@ -26,8 +49,13 @@ public static void EmbedCaptionsOnCurrentSlide()
             var currentSlide = PowerPointCurrentPresentationInfo.CurrentSlide;
             if (currentSlide != null)
             {
-                RemoveCaptionsFromSlide(currentSlide);
-                EmbedCaptionsOnSlide(currentSlide);
+                EmbedCaptionsOnSlides(
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/NotesToCaptions.cs]: no need RemoveCaptionsFromSlide?

**Comment by kai33** [comment]
Hi Kenneth, can you also add FT to guard the new changes? thanks

File: PowerPointLabs/PowerPointLabs/NotesToCaptions.cs (hunk 1 above)
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/NotesToCaptions.cs]: oh included

**Comment by Kenneth-LJS** [comment]
@kai33 Added test for no notes on the slide. However, the testing framework has no support for deselecting slides and selecting multiple slides. The testing framework has to be modified to support these operations, which can be addressed in future issues.

**Pull request merged**